app/src/main/java/com/eveningoutpost/dexdrip/Services/SnoozeOnNotificationDismissService.java

```diff
@@ -42,4 +60,10 @@ protected void onHandleIntent(Intent intent) {
 
         AlertPlayer.getPlayer().Snooze(getApplicationContext(), snooze);
     }
+    
+    private void snoozeOtherAlert(String alertType) {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
+        int snoozeMinutes = MissedReadingService.readPerfsInt(prefs, "other_alerts_snooze", 20);
+        UserNotification.snoozeAlert(alertType, snoozeMinutes);
```

Comment:
Shouldn't this be only called when the reraise feature is activated?

Scenario: Reraise feature is deactivated as the user does just want an unintrusive Notification sound; Snooze is set to 30 minutes.
At time 0 a predictive alert happens. At time 25 the user looks at the phone and swipes away the notification.
What happens at time 30?
I believe with the current logic a new predictive alert would rise. After the PR it would still be snoozed until time 55.